app/zadanieDnia.js

```diff
@@ -1 +1,26 @@
-//Twój kod
\ No newline at end of file
+//Twój kod
+const fs = require('fs');
+
+const defaultFilePath = './data/zadanieDnia/test.txt';
+const filePath = process.argv[2];
+const outputPath = './data/zadanieDnia/output.txt'
+
+const setCase = (letter, index) => {
```

Comment:
Bardzo fajny podział na osobne funkcje!

Reply:
No właśnie mam wrażenie ze setCase jest już niepotrzebny przy zoptymalizowaniu kodu do jednego wiersza. 

Bardziej jestem niezadowolony z pętli for, która próbowałem zastąpić: text.split().forEach((letter, index) => ...itd, ale nie wiem czemu otrzymywałem pod letter cały tekst. Czy split(’’) zadziała?

Reply:
Tak, `split('')`, bo "If separator is omitted, the array returned contains one element consisting of the entire string. If separator is an empty string, str is converted to an array of characters." (https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/String/split)